docs/go-run-your-first-app.md

```diff
@@ -5,36 +5,40 @@ sidebar_label: Run your first app
 ---
 import { ResponsivePlayer } from '../src/components'
 
-Welcome to the evolution of application development!
+<img class="docs-image-centered" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/rocket-launch-go.jpg" />
 
-This is a tutorial for developers who are new to [Temporal](/docs/overview) and have some basic knowledge of Go. We recommend setting aside ~20 minutes to complete it. By following this tutorial, you will achieve a few things:
+:::note Tutorial information
+- **Level**: ⭐ Temporal beginner
+- **Time**: ⏱️ ~20 minutes
+- **Goals**: 🙌
+  - Complete several runs of a Temporal Workflow application using the Temporal server and the [Go SDK](https://github.com/temporalio/go-sdk).
+  - Practice accessing and using the visibility of the Workflow's state.
```

Comment:
You don't really use visibility

Reply:
To be clear I mostly have an issue with wording

Reply:
What about?
- Practice reviewing the state of the Workflow.